app/api/author/index.ts

```diff
@@ -25,6 +25,7 @@ export interface ConnectAuthorParams {
   email: string;
   name: string;
   affiliationId: number | null;
+  affiliationName: string | null;
```

Comment:
This variable is required. I wonder why you have specified this as a null type.

Reply:
Thanks